nodejs/BUILD.bazel

```diff
@@ -36,3 +37,12 @@ toolchain_type(
     )
     for key, values in PLATFORMS.items()
 ]
+
+# User set flags that are passed to the build
+# bazel test --//nodejs:default_args="" //...
+# allowed args ["--preserve-symlinks", ""]
+user_args(
+    name = "default_args",
+    build_setting_default = "--preserve-symlinks",
```

Comment:
if this is the default, then why did you have to add it explicitly to a bunch of examples? seems wrong, I'd expect the examples not to change, since those illustrate the changes users would have to make as well.

Reply:
The flags were added to the examples to see how things would work when the default was changed. If we wanted to go with the nodejs default. As mentioned earlier in the comments on the pr. I'm happy to remove them from this pr though and reevaluate when we want to change the default.

Reply:
I think keeping the default to match what node_repositories() had makes sense for 5.0. We'd want to do some more due diligence before we flip the default.